lutris/gui/config/boxes.py

```diff
@@ -732,7 +732,7 @@ def on_reset_button_clicked(self, btn, option, _widget, wrapper):
     def set_style_property(property_, value, wrapper):
         """Add custom style."""
         style_provider = Gtk.CssProvider()
-        style_provider.load_from_data("GtkHBox {{{}: {};}}".format(property_, value).encode())
+        style_provider.load_from_data(f"GtkHBox {{{property_}: {value};}}".encode())
```

Comment:
Hang on, does this work? Don't you have to double up the curly braces that do not signify interpolation?

I would prefer this line be left as a ``.format`` call just to avoid that business all together.

Reply:
OK, I've tested this, and your code works. The doubled {} are already doubled in the original version, so I guess this is fine.